### ISSUE TITLE
In Android14 device, autofilling is not performed after receiving SMS messages.

### ISSUE THREAD

**Issue by acn-masatadakurihara** [andreszs/cordova-plugin-sms-retriever#12]
In targetSDK34+Android14 environment, autofilling is not performed after receiving SMS messages.

As per the following documentation, it appears that for apps targeting Android 14, a flag needs to be set indicating whether the receiver should be exported to all other apps on the device.
either `RECEIVER_EXPORTED` or `RECEIVER_NOT_EXPORTED`.
https://developer.android.com/about/versions/14/behavior-changes-14#runtime-receivers-exported

Specifically, [this line](https://github.com/andreszs/cordova-plugin-sms-retriever/blob/a8fa6b5926d7d546237fa4f9d0adecbb9d34998a/src/android/SMSRetriever.java#L155) should be changed as follows
```
applicationContext.registerReceiver(smsBroadcastReceiver, intentFilter);
```
to
```
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
  applicationContext.registerReceiver(smsBroadcastReceiver, intentFilter, Context.RECEIVER_EXPORTED);
} else {
  applicationContext.registerReceiver(smsBroadcastReceiver, intentFilter);
}
```

I have confirmed that the above modifications work correctly.
Please confirm.

**Comment by MaximBelov** [comment]
Please create PR

**Comment by acn-masatadakurihara** [comment]
@MaximBelov 
Please confirm #13

**Comment by andreszs** [comment]
Thanks, your PR #13 has been merged and updated 4.0.1 to make it backwards compatible with min API level 21 and target 31 again. Feel free to test it in Android 5.1 onwards.
